Source/WebCore/style/StyleExtractorConverter.h

```diff
@@ -592,7 +593,7 @@ inline RefPtr<CSSValue> ExtractorConverter::convertTransformOperation(const Rend
     case TransformOperation::Type::RotateX:
         return CSSFunctionValue::create(CSSValueRotateX, CSSPrimitiveValue::create(uncheckedDowncast<RotateTransformOperation>(operation).angle(), CSSUnitType::CSS_DEG));
     case TransformOperation::Type::RotateY:
-        return CSSFunctionValue::create(CSSValueRotateX, CSSPrimitiveValue::create(uncheckedDowncast<RotateTransformOperation>(operation).angle(), CSSUnitType::CSS_DEG));
+        return CSSFunctionValue::create(CSSValueRotateY, CSSPrimitiveValue::create(uncheckedDowncast<RotateTransformOperation>(operation).angle(), CSSUnitType::CSS_DEG));
```

Comment:
This deserves a test case. This was found by the "check direct serialization against value serialization mode".